views/cypress/tests/import-export.spec.js

```diff
@@ -63,19 +65,31 @@ describe('Import/export items', () => {
                 selectors.deleteClassUrl,
                 true
             );
-        };
-
-        it('can import item', function () {
-            importItemTest('e2e_item.zip');
         });
 
-        it('can import item with shared stimulus', function () {
-            importItemTest('e2e_item_shared_stimulus.zip');
+        [{
+            format: null, // QTI/APIP Content Package
+            filename: 'e2e_test_item.zip',
+        },{
+            format: 'QTI/APIP XML Item Document',
+            filename: 'e2e_test_item.xml'
+        },{
+            format: 'RDF',
+            filename: 'e2e_test_item.rdf'
+        }].forEach((testcase, index) => {
+            it(`${index}: "Import - ${testcase.format || 'QTI/APIP Content Package'}"`, function () {
+                cy.selectNode(selectors.root, selectors.itemClassForm, className);
+                cy.importToSelectedClass(selectors.importItem, `${packagesPath}/${testcase.filename}`, selectors.importItemUrl, className, testcase.format);
+                cy.wait(50); // Safe delay for working with FS in Cypress
```

Comment:
https://github.com/oat-sa/extension-tao-item/pull/572/files#r814812207

If it depends on the environment, we might consider provide this timeout through configuration